src/ResourceManagement/Compute/Microsoft.Azure.Management.Compute.Fluent/VirtualMachineCustomImagesImpl.cs

```diff
@@ -46,23 +47,17 @@ public IBlank Define(string name)
         }
 
         ///GENMHASH:7D6013E8B95E991005ED921F493EFCE4:6FB4EA69673E1D8A74E1418EB52BB9FE
-        public PagedList<IVirtualMachineCustomImage> List()
+        public IEnumerable<IVirtualMachineCustomImage> List()
         {
-            var pagedList = new PagedList<ImageInner>(Inner.List(), (string nextPageLink) =>
-            {
-                return Inner.ListNext(nextPageLink);
-            });
-            return WrapList(pagedList);
+            return WrapList(Inner.List()
+                .AsContinuousCollection(link => Inner.ListNext(link)));
         }
 
         ///GENMHASH:95834C6C7DA388E666B705A62A7D02BF:F27988875BD81EE531DA23D26C675612
-        public PagedList<IVirtualMachineCustomImage> ListByGroup(string resourceGroupName)
+        public IEnumerable<IVirtualMachineCustomImage> ListByGroup(string resourceGroupName)
         {
```

Comment:
```C#
IEnumerable<ResourceT> resourceTs = Inner.ListAll()
     .AsContinuousCollection(link => Inner.ListAllNext(link)));
```
Does the above line of code kickoff a call to fetch the first page? 

A fully LAZY IEnumerable<ResourceT> starts any operation only when user starts iterating through the list. i.e. following code (1) will not fetch the first page

(1)
```C#
IEnumerable<ResourceT> resourceTs = Inner.ListAll()
    .AsContinuousCollection(link => Inner.ListAllNext(link)));
```
But when user do foreach on enuemrable (2) then only any api call happens.

(2)
```C#
foreach(ResourceT vm : virtualMachines) {
  ...
}
```